libs/wb-types/src/weapon.ts

```diff
@@ -42,17 +42,17 @@ export enum Weapon {
     SCAR = "p93",
     TacticalShotgun = "p94",
     VEK = "p95",
-    UNKNOWN_WEAPON_96 = "p96",
-    UNKNOWN_WEAPON_97 = "p97",
+    Desert = "p96",
+    Auto = "p97",
```

Comment:
The weapon name needs to be more specific than `Auto` and `Desert`

Reply:
https://discord.com/channels/989707899519524924/989707899519524927/1314345792764641280

They are what they're called in-game, so we're following that